source/algorithms.tex

```diff
@@ -8345,7 +8433,7 @@
 
```

Comment:
Pre-existing: I've never been really happy with the handwavy "is no greater than" and "is less than" in the spec of this algorithm; adding projections to the mix exacerbates the problem. Consider rewording:
```
\pnum
Let tcode{proj} be \tcode{identity\{\}} 
for the overloads with no parameter \tcode{proj}, 
and let \tcode{comp} be \tcode{less\{\}} 
for the overloads with no parameter \tcode{comp}.

\pnum
\expects
\tcode{bool(comp(proj(hi), proj(lo)))} is \tcode{false}.
For the first form, type \tcode{T}
meets the \oldconcept{LessThan\-Comparable}
requirements (\tref{cpp17.lessthancomparable}).

\pnum
\returns
\tcode{lo} if \tcode{bool(comp(proj(v), proj(lo)))} is \tcode{true},
\tcode{hi} if \tcode{bool(comp(proj(hi), proj(v)))} is \tcode{true},
otherwise \tcode{v}.
```

Reply:
(I've pulled this out into #3785, please resolve this comment since I can't.)